src/dev-app/datepicker/datepicker-demo.ts

```diff
@@ -42,15 +42,20 @@ export class DatepickerDemo {
   minDate: Date;
   maxDate: Date;
   startAt: Date;
-  date: Date;
+  date: any;
   lastDateInput: Date | null;
   lastDateChange: Date | null;
   color: ThemePalette;
 
   dateCtrl = new FormControl();
 
-  dateFilter =
-      (date: Date) => !(date.getFullYear() % 2) && (date.getMonth() % 2) && !(date.getDate() % 2)
+  dateFilter: (date: Date | null) => boolean =
+    (date: Date | null) => {
+      if (date === null) {
```

Comment:
Maybe just change the one below to `!date || (...)`.

Reply:
I considered what Paul had in https://github.com/angular/components/pull/16373/files#diff-72d99d967518ce37f1e422881c29a43aR60, but it seemed to be less readable.

Your suggestion is?
```ts
return !date || !(date.getFullYear() % 2) && Boolean(date.getMonth() % 2) && !(date.getDate() % 2);
```

Reply:
Yes, exactly. It's a nit so feel free to ignore.